### ISSUE TITLE
Why calling create_output_panel twice?

### ISSUE THREAD

**Issue by deathaxe** [tomv564/LSP#21]
Why do you call `window.create_output_panel()` after applying settings again? It is the first time, I see something like that and did not realize any issue with not doing so. I am running ST 4142. Is it a workaround for earlier builts?

```python
def create_references_panel(window: sublime.Window):
    panel = window.create_output_panel("references")
    base_dir = get_project_path(window)
    panel.settings().set("result_file_regex",
                         r"^\s+(\S*)\s+([0-9]+):?([0-9]+)$")
    panel.settings().set("result_base_dir", base_dir)
    panel.settings().set("line_numbers", False)
    panel.assign_syntax("Packages/" + PLUGIN_NAME +
                        "/Syntaxes/References.sublime-syntax")

    # call a second time to apply settings   ??? really needed ???
    return window.create_output_panel("references")
```

**Comment by tomv564** [comment]
No idea where I picked that up from! If it was a ST bug then it's likely fixed, let's remove it.

**Comment by tomv564** [comment]
Fixed in #20 by @deathaxe, thanks!
